test/datahike/norm/resources/naming-and-sorting-test/checksums.edn

```diff
@@ -0,0 +1,12 @@
+{:001-a1-example.edn
+ "b28aa2247b363cb556b4abba3593774b6c960d5958458bcf3ed46d86e355d563ea77c66b1f4a21cb35f232227a8d4747dd7aec519bab036c706670eb5cf4a05c",
```

Comment:
I personally would prefer hasch UUIDs because they are shorter and hence more readable and conflicts between them are still less likely than asteroid impacts on earth. But it is up to you. The most important thing is that these hashes are not platform or encoding dependent (hasch can operate on ASTs of Clojure, i.e. you could read in a whole file and get a hash for syntax that ignores whitespace and orderings in maps and hash maps).

Reply:
I used edn-hasch -> platform/hasch->str so it should be using the clojure ast, right?

Reply:
Yes, nice!